app/assets/main.scss

```diff
@@ -6,17 +6,39 @@ $govuk-assets-path: "/static/assets/";
 @import "../../node_modules/govuk-frontend/dist/govuk";
 
 .app-\!-no-wrap {
-    white-space: nowrap;
```

Comment:
🤏 very minor nit but this changes the tab spacing to 2 where in most files we use 4, we should be consistent but not doing that would also help focus the review! We should get these formatted alongside HTML I'll go pick that PR back up soon